app/controllers/application_controller.rb

```diff
@@ -14,6 +14,22 @@ class ApplicationController < ActionController::Base
     redirect_to(manuals_path, flash: { error: "Manual not found" })
   end
 
+  def preview_design_system?(next_release: false)
+    # Temporarily force this flag to 'on' for users in production, regardless of their permissions
+    return true if next_release && !Rails.env.test?
+
+    current_user.can_preview_design_system? || (next_release && current_user.can_preview_next_release?)
+  end
+  helper_method :preview_design_system?
+
+  def render_design_system(design_system_view, legacy_view, locals)
+    if get_layout == "design_system"
+      render(design_system_view, locals:)
+    else
+      render legacy_view, locals:
+    end
+  end
```

Comment:
Is this all copied and pasted from Whitehall? I was quite confused by the need for a temporary forcing - do we actually need to temporarily force anything? I'm guessing Whitehall reasons for temporary forcing something and this apps might differ?

Also seems a bit strange to be adding this render_design_system code with a get_layout method when nothing calls the render_design_system method and get_layout is not defined anywhere (as an aside, can we avoid a `get_layout` method it's not very idiomatic Ruby to have a getter)